### PR TITLE
🎉  Single py checker

### LINKED ISSUE #9612: 🎉  Detect changes of CDK and SAT folders for SonarQube
Now the SQ workflow is detecting all changes for connectors into the folder `./airbyte-integrations/connectors` only. But we need to detect and all other Python packages/components too.
All details of the current implementation are [here](https://github.com/airbytehq/airbyte/pull/8362)

### LINKED ISSUE #9750: 🐛  Single config of Python linters
Now there are several points where checkers can be launched:
- with ` ./gradlew  format`
- with SonarQube script
- with some custom developer scripts

The second critical problem is results of launched tools can be conflicted. 

## Expected Behavior
use single script for all tools with the single config(pyproject.toml)

### PR DESCRIPTION
**Pull request by antixar**
## What
Different scripts use different Python checkers' config files and versions. 

## How
Correct all checker settings and setup the single `pyproject.toml` file for all scripts

### CODE REVIEW

File: airbyte-integrations/connectors/source-strava/source_strava/source.py [hunk 1]
```diff
@@ -4,16 +4,22 @@
...
+
+
+
+
 
 # Source
 class SourceStrava(AbstractSource):
     def check_connection(self, logger, config) -> Tuple[bool, any]:
         """
+        dddd = "aaaaaaaa"
```
Comment (grubberr) [inline airbyte-integrations/connectors/source-strava/source_strava/source.py:22]: can be removed ?

File: tools/code-generator/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+
+RUN apk --no-cache upgrade \
+    && pip install --upgrade pip  \
+    && apk add  g++ make
+
+# the new version (>= 2.1.0) of package markupsafe removed the funcion `soft_unicode`. And it broke other dependences
+# https://github.com/pallets/markupsafe/blob/main/CHANGES.rst
+# thus this version is pinned
+RUN  pip install --prefix=/install markupsafe==2.0.1
+RUN  pip install --prefix=/install black==22.1.0 datamodel_code_generator==0.11.19
```
Comment (grubberr) [inline tools/code-generator/Dockerfile:13]: as I understand new markupsafe==2.1.0 incopatible with old datamodel-code-generator==0.10.1

but it's ok to use:
```
markupsafe==2.1.0
datamodel_code_generator==0.11.19
```

I mean we can just upgrade `datamodel_code_generator` and don't touch `markupsafe` at all ?

**Review by grubberr — APPROVED** [review]

File: airbyte-integrations/connectors/source-strava/source_strava/source.py (hunk 1 above)
Comment (antixar) [reply airbyte-integrations/connectors/source-strava/source_strava/source.py:22]: yes, it is test line for checking

File: tools/code-generator/Dockerfile (hunk 2 above)
Comment (antixar) [reply tools/code-generator/Dockerfile:13]: you are right. Fixed

**Pull request merged**